front/src/applications/operationalStudies/components/Project/AddOrEditProjectModal.tsx

```diff
@@ -102,7 +95,7 @@ export default function AddOrEditProjectModal({ editionMode, project, getProject
       try {
         let imageId = currentProject.image;
         if (tempProjectImage) {
-          imageId = await getDocKey(tempProjectImage as Blob);
+          imageId = await postDocument(tempProjectImage as Blob);
         } else if (tempProjectImage === null) {
           imageId = null;
           setTempProjectImage(imageId);
```

Comment:
the result is:
![image](https://user-images.githubusercontent.com/20640252/233581844-8ae9b00c-efad-40f5-b088-d9abc6f7ddf2.png)
maybe we should not talk about image**Id** ?

Reply:
No, you are looking at the response of the GET /document/, not the one of POST /document/ :) 
![image](https://user-images.githubusercontent.com/45000526/233667886-abae7642-5155-4ef6-b409-91ff7442f100.png)

Reply:
I was looking at tempProjectImage, I thought `setTempProjectImage(null)` could be better for good understanding but it's ok